assets/css/sass/utils/_mixins.scss

```diff
@@ -32,6 +32,15 @@
 	}
 }
 
+// Remove underline if element has a button class.
+@mixin notUnderlinedWhenButton() {
```

Comment:
I wonder if we can make this name clearer - starting with `not` always confuses me. Not a blocker but if you come up with something clearer before merge go for it :)

Does this remove underline from buttons? If so, we could call this `removeButtonUnderline` or similar.

Reply:
Yeah, much better name. Renamed in 97894ccfa67b3367a7535117b89afb7a1e978f6d.